### DISCUSSION TITLE
Rust Embed example

### DISCUSSION THREAD

**Discussion by josejachuf** [salvo-rs/salvo#257] [Q&A]
Hello
I am trying to use Rust Embed as the example shows, but I have a problem with the routes.

this works fine:
```
    let router = Router::with_path("<**path>")
            .get(static_embed::<Assets>().with_fallback("index.html"))
    ;
```

but if I change the route to 
`with_path("app/<**path>") `
only load the index.html, not the other files, show a blank page.

```
    let router = Router::new().get(hello_world)
        .push(
            Router::with_path("app/<**path>")
            .get(static_embed::<Assets>().with_fallback("index.html")),
    );
```

```
<script type="module" crossorigin src="/assets/index-28a48a45.js"></script>
<link rel="stylesheet" href="/assets/index-81e4655a.css">
```

These files are looking for them in:

http://127.0.0.1:5800/assets/index-28a48a45.js
and
http://127.0.0.1:5800/assets/index-81e4655a.css

Or is this the expected behavior?

Any ideas?
Thanks

**Comment by chrislearn** [comment]
Please give a demo repository to reproduce this bug.
